Hearthstone Deck Tracker/OpponentDeadForTracker.cs

```diff
@@ -0,0 +1,60 @@
+﻿using HearthDb.Enums;
+using Hearthstone_Deck_Tracker.Hearthstone;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Hearthstone_Deck_Tracker
+{
+	public static class OpponentDeadForTracker
+	{
+		private static List<string> _uniqueDeadHeroes = new List<string>();
+		private static List<int> _deadTracker = new List<int>();
+		const string KelThuzadCardId = "KelThuzad";
+
+		public static async void ShoppingStarted(GameV2 game)
+		{
+			if(game.GetTurnNumber() <= 1)
+			{
+				_uniqueDeadHeroes.Clear();
+				_deadTracker.Clear();
+				Core.Overlay.nextOpponentLeaderboardPosition = null;
```

Comment:
Related to the comment below, calling something like `Core.Overlay.ClearPositionDeadForText()` would be better than setting a field to null.